tests/integration_tests/test_external_database_connections.py

```diff
@@ -106,3 +106,22 @@ def test_snowflake_example_data(snowflake_test_engine):
         "service_request",
         "visit",
     ]
+
+@pytest.mark.integration_external
+@pytest.mark.integration_bigquery
```

Comment:
Are there any other tests that we should mark with the `integration_bigquery` annotation?

Reply:
Not in this PR- in the BigQuery query generation there are more.

notice that https://github.com/ethyca/fidesops/pull/189/files#diff-0fa654d3dbcc4abd3ae1889caf2316f7efa649d6bb7ec77abe6261fd16d35d6bR689 doesn't actually connect to a bigquery instance, so we won't need to mark it as an integration test